pegasus/sites.v3/code.org/public/private/edit-csp-properties/splat.haml

```diff
@@ -0,0 +1,124 @@
+-dont_cache
+
+-encrypted_app_id = request.splat_path_info[1..-1]
+-app_id = storage_decrypt_app_id(encrypted_app_id).last.to_s
+-api_base_url = "/v3/apps/#{encrypted_app_id}/shared-properties"
+
+-if rack_env == :development
```

Comment:
in pegasus there's a theme for including all the angular stuff, see:

https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml